requirements/extras.txt

```diff
@@ -0,0 +1,3 @@
+deepnog>=1.0.1
+torch>=1.2
```

Comment:
`torch` should better be handled as a transitive dependency of deepnog. I'd remove it here.